src/lib/DataTable/DataTableRouted.tsx

```diff
@@ -29,14 +29,15 @@ export function DataTableRouted<T, TFilter, TRouteNames>({
   handlers,
   tableClassName,
   tableStyle,
+  asc,
 }: DataTableRoutedProps<T, TFilter, TRouteNames>) {
   const [queryResult, setQueryResult] = useState<TableQueryResult<T>>(data);
   const [filterState, setFilterState] = useState<FilterPageState>({
     currentPage: 1,
     filter: predefinedFilter ?? {},
     itemsPerPage: predefinedItemsPerPage ?? 25,
   });
-  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc: true });
+  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc: asc });
```

Comment:
You can use the shortcut here because the names are the same:
```
  const [orderState, setOrderState] = useState<OrderOption>({ orderBy: columns[0].dataField, asc });
```

Not sure why prettier doesn't complain about it...